climate_index_collection/indices.py

```diff
@@ -46,6 +46,51 @@ def southern_annular_mode(data_set, slp_name="sea-level-pressure"):
     return SAM_index
 
 
+def southern_annular_mode_pc(data_set, slp_name="sea-level-pressure"):
```

Comment:
The problem with `SLP` is that it's not defined over most of the land. The values in the model data are likely extrapolated. So we should use Z500 here, I think.

Reply:
I prefer to stick to ```SLP``` but don't take *complete* Southern Hemisphere but only limit latitude to 40-65°S, since regular SAM index is defined as slp difference between these edges.

Reply:
Alright, I now agree and modified index definition accordingly: PC-based SAM is calculated on geopotential height anomalies. Please review and merge, to close #78 . I will then take care of #80 in a new pull request for *ALL* pc-based indices.